db/schema.rb

```diff
@@ -167,7 +167,6 @@
     t.string "distributor"
     t.string "documentation_url"
     t.string "support_url"
-    t.string "service_offering_icon_ref"
```

Comment:
That's odd, why did this get removed with this PR?

Reply:
After I run `db:migrate:reset`, it comes back.